src/cmt_encode_opentelemetry.c

```diff
@@ -2498,7 +2531,8 @@ static cfl_sds_t render_opentelemetry_context_to_sds(
     return result_buffer;
 }
 
-cfl_sds_t cmt_encode_opentelemetry_create(struct cmt *cmt)
+cfl_sds_t cmt_encode_opentelemetry_create_with_cutoff_opts(struct cmt *cmt,
+                                                           struct cmt_opentelemetry_context_cutoff_opts *opts)
```

Comment:
I think the options should be generic for the encoder and not only associated to cutoff functionality, e.g:

```c

struct cmt_encode_opentelemetry_opts {
    int      cutoff_enabled;
    int64_t  cutoff_threshold;
};

cfl_sds_t cmt_encode_opentelemetry_create(struct cmt *cmt, 
                                          struct cmt_encode_opentelemetry_opts *opts)
```

Reply:
Sure, I fixed it in https://github.com/fluent/cmetrics/pull/223/commits/92a16b9c094fa325ef49ed8a36364c9123a4464e.